src/lib/structures/ExtendedArgument.ts

```diff
@@ -5,36 +5,36 @@ import { Argument, ArgumentContext, ArgumentOptions, ArgumentResult, AsyncArgume
 
 /**
  * The extended argument class. This class is abstract and is to be extended by subclasses which
- * will implement the [[ExtendedArgument#handle]] method.
- * Much like the [[Argument]] class, this class handles parsing user-specified command arguments
+ * will implement the {@link ExtendedArgument#handle} method.
+ * Much like the {@link Argument} class, this class handles parsing user-specified command arguments
  * into typed command parameters. However, this class can be used to expand upon an existing
  * argument in order to process its transformed value rather than just the argument string.
  *
  * @example
  * ```typescript
  * // TypeScript:
  * import { ApplyOptions } from '@sapphire/decorators';
- * import { ArgumentResult, ExtendedArgument, ExtendedArgumentContext, ExtendedArgumentOptions } from '(at)sapphire/framework';
+ * import { ArgumentResult, ExtendedArgument, ExtendedArgumentContext, ExtendedArgumentOptions } from '@sapphire/framework';
  * import type { Channel, TextChannel } from 'discord.js';
  *
  * // Just like with `Argument`, you can use `export default` or `export =` too.
- * @ApplyOptions<ExtendedArgumentOptions>({
+ * (at)ApplyOptions<ExtendedArgumentOptions>({
  *   name: 'textChannel',
  *   baseArgument: 'channel'
  * })
  * export class TextChannelArgument extends ExtendedArgument<'channel', TextChannel> {
  *   public handle(parsed: Channel, { argument }: ExtendedArgumentContext): ArgumentResult<TextChannel> {
  *     return parsed.type === 'text'
  *       ? this.ok(parsed as TextChannel)
- *       : this.error(argument, 'ArgumentTextChannelInvalidTextChannel', 'The argument did not resolve to a text channel.');
+ *       : this.error({ identifier: 'ArgumentTextChannelInvalidTextChannel', message: 'The argument did not resolve to a text channel.' });
```

Comment:
Too long, please format this line with Prettier so it fits within the character count.